pkg/ibmcsidriver/server.go

```diff
@@ -114,13 +126,22 @@ func (s *nonBlockingGRPCServer) Setup(endpoint string, ids csi.IdentityServer, c
 
 	s.logger.Info("Start listening GRPC Server", zap.Reflect("Scheme", u.Scheme), zap.Reflect("Addr", addr))
 
+	// Create listener
 	listener, err := net.Listen(u.Scheme, addr)
 	if err != nil {
 		msg := "failed to listen GRPC Server"
 		s.logger.Error(msg, zap.Reflect("Error", err))
 		return nil, errors.New(msg)
 	}
 
+	if os.Getenv("IS_NODE_SERVER") == "true" {
+		fileops := &realFileOps{}
+		if err := setupSidecar(addr, fileops); err != nil {
+			s.logger.Error("setupSidecar failed.", zap.Error(err))
+			return nil, err
```

Comment:
@prankulmahajan in case of any error on this setup the driver wont be running ?

Reply:
Yes container will crash